docs/source/analysis.rst

```diff
@@ -13,6 +13,7 @@ Functions
   compute_entropy_gain <_autosummary/dynsight.analysis.compute_entropy_gain>
   compute_shannon_multi <_autosummary/dynsight.analysis.compute_shannon_multi>
   compute_entropy_gain_multi <_autosummary/dynsight.analysis.compute_entropy_gain_multi>
+  sample_entropy <_autosummary/dynsight.analysis.sample_entropy>
   compute_sample_entropy <_autosummary/dynsight.analysis.compute_sample_entropy>
```

Comment:
Change name to `combine_sample_entropies`?? Or something about the math you are doing to do this.